pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -82,11 +83,15 @@ func (hm *hostportManager) Add(id string, podPortMapping *PodPortMapping, natInt
 		return nil
 	}
 
-	if podPortMapping.IP.To4() == nil {
+	if podPortMapping.IP.To16() == nil {
 		return fmt.Errorf("invalid or missing IP of pod %s", podFullName)
 	}
 	podIP := podPortMapping.IP.String()
 
+	if utilnet.IsIPv6String(podIP) != hm.iptables.IsIpv6() {
```

Comment:
with multiple pod IPs, should this be scanning the list looking for any match?  @khenidak